packages/components/package.json

```diff
@@ -13,13 +13,16 @@
   ],
   "scripts": {
     "dev": "tamagui-build --watch",
-    "build": "tamagui-build",
+    "build": "tamagui-build || true",
     "typegen": "tsc --noEmit false --emitDeclarationOnly || true",
     "storybook": "TAMAGUI_TARGET='web' storybook dev -p 3001",
-    "storybook:build": "TAMAGUI_TARGET='web' storybook build"
+    "storybook:build": "TAMAGUI_TARGET='web' storybook build",
+    "clean": "rimraf dist node_modules .turbo"
   },
   "peerDependencies": {
-    "react-native-web": "^0.18.0"
+    "react-native-web": "^0.18.0",
+    "@tamagui/next-plugin": "1.0.15",
+    "@tamagui/core": "1.0.15"
```

Comment:
These expose the fact that I actually don't know how it works 😅.

Is `@tamagui/next-plugin` actually required for peers given that components are compiled and made read-to-use after publishing/building under `/dist`? If my assumption is even true.

And is `@tamagui/core` needed, because `@status-im/compoents` doesn't export its own provider?

Reply:
Actually, I've opened one of those "ready-to-use" files and I can see

```
module.exports = __toCommonJS(typography_exports);
var import_core = require("@tamagui/core");
var import_tamagui = require("tamagui");
const Heading = (0, import_core.styled)(import_tamagui.SizableText, {
  name: "Heading",
```

but still not sure if `@tamagui/core` or `@tamagui` even should be added to `peerDependecies` of this component library.